pkgs/applications/misc/openring/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoModule, fetchFromSourcehut, lib }:
+buildGoModule rec {
+  pname = "openring";
+  version = "master";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = pname;
+    rev = "f13edb5dfd882ce608d61cf6b6740650ce9d84a3";
+    sha256 = "sha256-Z65V77JZ9jCzBg7T2+d5Agxxd+MV2R7nYcLedYP5eOE=";
+  };
+
+  vendorSha256 = "sha256-BbBTmkGyLrIWphXC+dBaHaVzHuXRZ+4N/Jt2k3nF7Z4=";
+
+  doCheck = false;
```

Comment:
Why are tests disabled?

Reply:
Two reasons, (1) I copied from a different package that had this set to false, and (2) there aren't any tests in the upstream package.

Reply:
Should I just remove this, even though there are no tests?

Reply:
Adding a comment that the package has not tests is fine.